CFP.md

```diff
@@ -7,15 +7,15 @@ layout: page
 ### Call for papers for DEF CON 28 opening soon!
 We want to give you a platform to talk on!  Have a cool hack you want to show off?  Want to teach something awesome?  Feel like blabbering on in front of a crowd?  Let us know!
 
-We're most interested in the following:
+This year more than others we  are seeking anything and everything related to making your own use that is hands-on. Do you have a training or workshop that includes a hands-on component? Perhaps there is a specific meetup of a special (hardware) interest group you would like to lead? Topics that we are always interested in include:
```

Comment:
Looks like a double space between "we" and "are." I don't think this shows up on the final webpage, but if so, it should be fixed.

Reply:
good eye